README.md

```diff
@@ -197,20 +197,12 @@ new ForkTsCheckerWebpackPlugin({
 - **memoryLimit** `number`:
   Memory limit for service process in MB. If service exits with allocation failed error, increase this number. Default: `2048`.
 
-- **workers** `number`:
-  You can split type checking to a few workers to speed-up increment build. **Be careful** - if you don't want to increase build time, you
-  should keep free 1 core for _build_ and 1 core for a _system_ _(for example system with 4 CPUs should use max 2 workers)_. Second thing -
-  node doesn't share memory between workers - keep in mind that memory usage will increase. Be aware that in some scenarios increasing workers
-  number **can increase checking time**. Default: `ForkTsCheckerWebpackPlugin.ONE_CPU`.
-
 - **vue** `boolean | { enabled: boolean, compiler: string }`:
   If `true` or `enabled: true`, the linter and compiler will process VueJs single-file-component (.vue) files. See the
   [Vue section](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin#vue) further down for information on how to correctly setup your project.
 
 - **useTypescriptIncrementalApi** `boolean`:
-  If true, the plugin will use incremental compilation API introduced in TypeScript 2.7. In this mode you can only have 1
-  worker, but if the changes in your code are small (like you normally have when you work in 'watch' mode), the compilation
-  may be much faster, even compared to multi-threaded compilation. Defaults to `true` when working with TypeScript 3+ and `false` when below 3. The default can be overridden by directly specifying a value.
+  If true, the plugin will use incremental compilation API introduced in TypeScript 2.7. Defaults to `true` when working with TypeScript 3+ and `false` when below 3. The default can be overridden by directly specifying a value.
```

Comment:
Could we introduce a comment here to indicate that `useTypescriptIncrementalApi` does not support Vue please? I thought we already had this but I was mistaken 😄

Reply:
Are you sure? I saw the code that handles `vue` in the `CompilerHost.ts` that is used only by the `ApiIncrementalChecker.ts`

Reply:
It worked kind-of but primarily not really.
This was what #231 was meant to tackle, but as I had not enough feedback (or a project to test it on) back then and no time to finish it right now, it's still open :/

Reply:
(see also #219, #220